src/components/CalculatorDisplay.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+const CalculatorDisplay = () => (
+  <>
+    <div className="container">
+      <div className="sum-display sum">
+        <span>0</span>
+      </div>
+
+      <button type="button" className="btn-white">
+        AC
+      </button>
+      <button type="button" className="btn-white">
+        +/-
+      </button>
+      <button type="button" className="btn-white">
+        %
+      </button>
+      <button type="button" className="btn-orange">
+        &#xf7;
+      </button>
+
+      <button type="button" className="btn-white">
+        7
+      </button>
+      <button type="button" className="btn-white">
+        8
+      </button>
+      <button type="button" className="btn-white">
+        9
+      </button>
+      <button type="button" className="btn-orange">
+        x
+      </button>
+
+      <button type="button" className="btn-white">
+        4
+      </button>
+      <button type="button" className="btn-white">
+        5
+      </button>
+      <button type="button" className="btn-white">
+        6
+      </button>
+      <button type="button" className="btn-orange">
+        -
+      </button>
+
+      <button type="button" className="btn-white">
+        1
+      </button>
+      <button type="button" className="btn-white">
+        2
+      </button>
+      <button type="button" className="btn-white">
+        3
+      </button>
+      <button type="button" className="btn-orange">
+        +
+      </button>
+
+      <button type="button" className="btn-white btn-zero">
+        0
+      </button>
+      <button type="button" className="btn-white">
+        .
+      </button>
+      <button type="button" className="btn-orange">
+        =
+      </button>
+    </div>
+  </>
+);
```

Comment:
- [ ] Great job implementing this function using the ES6 syntax but the requirement states that you should use functional components. Don't worry, there will be time to refactor your code to make it up to date with the industry trend.
![image](https://github.com/chiedozie013/math-magicians/assets/99715640/c929a1ae-2855-41d7-9c59-a4ee75af90fb)